warehouse/bigquery/bigquery.go

```diff
@@ -115,7 +115,9 @@ func (bq *HandleT) CreateTable(tableName string, columnMap map[string]string) (e
 		return
 	}
 
-	err = bq.createTableView(tableName, columnMap)
+	if !isDedupEnabled {
+		err = bq.createTableView(tableName, columnMap)
+	}
```

Comment:
for my understanding - excluding users table, customer needs to create views himself if the table was created when dedup is enabled and now switched to append method, right?

Reply:
That's right